game-and-author/game.rb

```diff
@@ -0,0 +1,10 @@
+class Game
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(multiplayer, last_played_at)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?()end
+end
```

Comment:
- [ ] Why do you have this game and author class in the class folder here and the same type?